### ISSUE TITLE
False positives from ErrorCause feature check

### ISSUE THREAD

**Issue by kitschpatrol** [yowainwright/es-check#240]
Thanks for this project!

Even though `new Error("message")` syntax has been valid since ES3, `es-check` with the `--checkFeatures` flag seems to mark this as requiring > ES13 regardless of whether a `cause` option is passed to the `Error` constructor.

For example:

Given file:
```js
// error-without-cause.js
throw new Error('Something went wrong')
```

We get this incorrect error from `es-check`:
```sh
$ es-check es10 ./error-without-cause.js --checkFeatures
...`Error: Unsupported features detected: ErrorCause. These require a higher ES version than 10.`
```

I took a quick look at the feature description code, and found that its `astInfo` has a `hasOptionsCause` value which I would assume might match against something in the AST:

```js
  ErrorCause: {
    minVersion: 13,
    example: 'new Error("...", { cause: e })',
    astInfo: {
      nodeType: 'NewExpression',
      callee: 'Error',
      hasOptionsCause: true,
    },
  },
```

...but I couldn't find a code path in `es-check` where this is ever actually verified, and never saw such a key in raw `acorn` output, though maybe I'm missing something!

**Comment by yowainwright** [comment]
Thanks for your comment @kitschpatrol! I will add this to a batch fix pretty soon if I can reproduce and test. 🙏

**Comment by yowainwright** [comment]
Re-opened. Thanks for the feedback! 🙇

**Comment by yowainwright** [comment]
@kitschpatrol can you check if [9.1.2-0](https://www.npmjs.com/package/es-check/v/9.1.2-0) fixed this issue? 
Thanks for your patience!

**Comment by kitschpatrol** [comment]
It's a fix, thanks!

```sh
echo "throw new Error('Something went wrong')" > test.js
npx es-check@9.1.2-0 es10 ./test.js --checkFeatures
info: ES-Check: there were no ES version matching errors!  🎉
```
